### DISCUSSION TITLE
Could not build wheels for oracledb, which is required to install pyproject.toml-based project

### DISCUSSION THREAD

**Discussion by avijit09** [oracle/python-oracledb#265] [Q&A]
I was trying to install a specific of oracledb==1.0.3 via Anaconda Prompt by running the following script

pip install oracledb==1.0.3

I have created a virtual environment where I was trying to install that specific package.

with the latest version I am getting this error "time zones are not supported in thin mode". Therefore, need to install 1.0.3

I m getting following error.,

Failed to build oracledb
ERROR: Could not build wheels for oracledb, which is required to install pyproject.toml-based projects

and

DPY-3022: named time zones are not supported in thin mode

I need to run it in thin client

**Comment by cjbj** [comment]
See https://github.com/oracle/python-oracledb/issues/264#issuecomment-1851038012
